### ISSUE TITLE
issues with talk npc

### ISSUE THREAD

**Issue by alisonrag** [OpenKore/openkore#1417]
will update the description in future, but the references is:
#2011 
#1187 
#1062 
#885 
#592

**Comment by syafiqrude** [comment]
bro , i have been trying to use the autosell but it didnt work , in the command it says like this
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
[config.txt](https://github.com/OpenKore/openkore/files/1485063/config.txt)
i have upload my config , mind telling me what should i do as im new in this .

**Comment by sctnightcore** [comment]
@syafiqrude  del you id PW !!!!

**Comment by syafiqrude** [comment]
@sctnightcore i have do exactly as u said , and i have no problem for autobuy , its just tht the autosell didnt work  and it still like this 
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96
Cannot calculate a route from payon (140,56) to 
Calculating auto-sell route to: (Payon): 159, 96

**Comment by sctnightcore** [comment]
well . 
route_randomWalk_inTown 1 ?

**Comment by syafiqrude** [comment]
@sctnightcore yeah , like i said , i have no problem for it to autobuy , just for the autosell tht i have problem , i have put it in item control all to 0 0 1 and it still cannot sell which is give me a headache haha . 
sellAuto 1
sellAuto_npc Payon 159 96
sellAuto_standpoint
sellAuto_distance 8
sellAuto_npc_steps

**Comment by sctnightcore** [comment]
https://github.com/OpenKore/openkore/blob/cb5be26f6c3cb5466c05ae9c3eac043a612cd46b/control/items_control.txt#L4-L11
@syafiqrude

**Comment by syafiqrude** [comment]
@sctnightcore can u give me the new link for new openkore ?

**Comment by syafiqrude** [comment]
@sctnightcore been doing the same as u guide , but still i dont know why it cannot calculate the route , its okay , guess have to do it manually , thanks for the help

**Comment by sctnightcore** [comment]
Download new openkore ! @syafiqrude  

And test it again !

**Comment by syafiqrude** [comment]
@sctnightcore i have download new one , setting up new config just like u ask me . the results still the same . 
[items_control.txt](https://github.com/OpenKore/openkore/files/1485411/items_control.txt)
the items control should be like this right ?

**Comment by syafiqrude** [comment]
the main problem is , it says 

> Cannot calculate a route from payon (159,94) to

**Comment by sctnightcore** [comment]
well, I tested everything and it worked.

![image](https://user-images.githubusercontent.com/23263315/32989418-f33e9fa8-cd47-11e7-9f1f-8279a89c8358.png)

**MENU > Program > Automatic Botting**

@alisonrag  come here !

**Comment by fadreus** [comment]
Use other NPC sell laa..
Aigoo..
XDDDDD

**Comment by sctnightcore** [comment]
@fadreus  I've tested it and it works.
I do not know what to do. His openkore does not work.

**Comment by fadreus** [comment]
@sctnightcore  Maybe if he can provide screenshot during autobuy & autosell we might know why.
Newbie tend to miss important information.
